packages/shared/src/lib/constants.ts

```diff
@@ -59,7 +59,8 @@ export const isChrome = (): boolean =>
 export const webappUrl = process.env.NEXT_PUBLIC_WEBAPP_URL;
 export const onboardingUrl = `${webappUrl}onboarding`;
 export const plusUrl = `${webappUrl}plus`;
-export const managePlusUrl = `${webappUrl}manage-plus`;
```

Comment:
Maybe we want this as a link via our redirector?

Reply:
Yeah though about it as well, let me ask Nimrod